docs/releases/4.1.md

```diff
@@ -80,3 +81,8 @@ The following button classes have been removed, none of which were being used wi
 * `hover-no`
 * `unbutton`
 * `yes`
+
+### Changed support for importing Spreadsheet files into Redirects
+
+* `.xls` files will no longer be supported for importing into the contrib Redirects listing, `.xlsx` is still supported.
+* `.tsv` files will now explicitly require the `.tsv` extension to be imported, `.csv` using tab delimiters will no longer work.
```

Comment:
I believe this was already true before my changes (I didn't verify)

Reply:
thanks.